docker-compose.yml

```diff
@@ -177,6 +177,9 @@ services:
       # Job Settings
       JOB_CACHE_TIMEOUT_DAYS: ${JOB_CACHE_TIMEOUT_DAYS:-30}
 
+      # VDI study access setting
+      VDI_SCHEMA_SUFFIX: ${VDI_SCHEMA_LIFECYCLE:-dev_n} # e.g. dev_n, prod_s, qa_n
```

Comment:
i think i like `VDI_SCHEMA_SUFFIX` better, because n/s is not a lifecycle

Reply:
Oops, will fix